src/Cake.Common/Tools/MSpec/MspecSettings.cs

```diff
@@ -94,12 +94,12 @@ public sealed class MSpecSettings : ToolSettings
         /// <summary>
         ///     Gets or sets a value indicating whether to enable reporting for AppVeyor CI integration(also auto - detected)
         /// </summary>
-        public bool Appveyor { get; set; }
+        public bool AppVeyor { get; set; }
 
         /// <summary>
         ///     Gets or sets a value indicating whether disable AppVeyor autodetection
         /// </summary>
-        public bool NoAppveyor { get; set; }
+        public bool NoAppVeyor { get; set; }
```

Comment:
Would this pair of breaking changes be accepted or should I drop them?

Reply:
@jnm2 Not sure. I suspect this will break a lot of things for a lot of people.

@cake-build/cake-team What do you think?

Reply:
Could obsolete with warning and have it backed by correct casing property and remove it next release.

Reply:
@devlead Yes, that's a great solution 👍

Reply:
Sounds good to me! Would you want to do anything like that for Convertable\*Path -> Convertible\*Path, which are entire classes? For compatibility, we'd need to have obsoleted classes with the existing names inheriting from new classes with proper spelling and all the functionality.

Reply:
I didn't hear back about renaming the classes from `Convertable` to `Convertible`, so I will leave them alone.